src/parse-text.js

```diff
@@ -0,0 +1,104 @@
+var moment = require('moment');
+
+function handleAMPM(text)
+{
+	var hours = parseInt(text.substring(0, text.search(':')), 10);
+	if ((/(AM|am)/).test(text))
+	{
+		if (hours === 12)
+		{
+			hours = 0;
+		}
+	}
+	else if ((/(PM|pm)/).test(text))
+	{
+		if (hours < 12)
+		{
+			hours += 12;
+		}
+	}
+	return hours;
+}
+
+function extractHoursMinutesFromString(text, searchStartIndex, searchEndIndex)
+{
+	var timeArray = (/(\w+:\w\w\s?(AM|am|PM|pm))/).exec(text.substring(searchStartIndex, searchEndIndex));
+	var timeString = `${ handleAMPM(timeArray[0]) }${ timeArray[0].substr(timeArray[0].search(':'), 3) }`;
+
+	if (timeString.length === 4)
+	{
+		timeString = `0${ timeString }`;
+	}
+	return timeString;
+}
+
+function checkIfTomorrow(text)
+{
+	if (text.includes('tomorrow') || text.includes('Tomorrow'))
```

Comment:
Couple things about this method:
1. Case-insenstitive string comparisons are less prone to break. For example, consider if I type `/wfh TOMORROW`, would this logic still work?
2. To make the method more compact, you can return the comparison directly as the result like `return text.toLowerCase().includes('tomorrow')`.